deployer/hub.py

```diff
@@ -439,14 +439,28 @@ def deploy(self, auth_provider, secret_key, skip_hub_health_test=False):
         """
         Deploy this hub
         """
+        if self.spec["auth0"]['enabled'] == False:
```

Comment:
I think we should read secret config file unconditionally if it exists, regardless of what auth0 is set to. This lets us put in other secret values (like a GCP service account JSON in environment, for example) without hassle.

Reply:
Done in 4f53a4c